File: C/1048.c
#include <stdio.h>
 
int main() {
 
    float a, reajuste;
    scanf("%f", &a);
    
    if(a <= 400.00){
        reajuste = (a * 15) / 100;
        printf("Novo salario: %.2f\n", a+reajuste);
        printf("Reajuste ganho: %.2f\n", reajuste);
        printf("Em percentual: 15 %\n");
    }else if(a > 400.00 && a <= 800.00){
        reajuste = (a * 12) / 100;
        printf("Novo salario: %.2f\n", a+reajuste);
        printf("Reajuste ganho: %.2f\n", reajuste);
        printf("Em percentual: 12 %\n");
    }else if(a > 800.00 && a <= 1200.00){
        reajuste = (a * 10) / 100;
        printf("Novo salario: %.2f\n", a+reajuste);
        printf("Reajuste ganho: %.2f\n", reajuste);
        printf("Em percentual: 10 %\n");
    }else if(a > 1200.00 && a <= 2000.00){
        reajuste = (a * 7) / 100;
        printf("Novo salario: %.2f\n", a+reajuste);
        printf("Reajuste ganho: %.2f\n", reajuste);
        printf("Em percentual: 7 %\n");
    }else{
        reajuste = (a * 4) / 100;
        printf("Novo salario: %.2f\n", a+reajuste);
        printf("Reajuste ganho: %.2f\n", reajuste);
        printf("Em percentual: 4 %\n");
    }
}

File: Python/1048.py
salario = float(input())

if salario > 0 and salario <= 400:
    reajuste = salario * 15 / 100
    print(f'Novo salario: {salario+reajuste:.2f}\nReajuste ganho: {reajuste:.2f}\nEm percentual: 15 %')
elif salario > 400 and salario <= 800:
    reajuste = salario * 12 / 100
    print(f'Novo salario: {salario+reajuste:.2f}\nReajuste ganho: {reajuste:.2f}\nEm percentual: 12 %')
elif salario > 800 and salario <= 1200:
    reajuste = salario * 10 / 100
    print(f'Novo salario: {salario+reajuste:.2f}\nReajuste ganho: {reajuste:.2f}\nEm percentual: 10 %')
elif salario > 1200 and salario <= 2000:
    reajuste = salario * 7 / 100
    print(f'Novo salario: {salario+reajuste:.2f}\nReajuste ganho: {reajuste:.2f}\nEm percentual: 7 %')
else:
    reajuste = salario * 4 / 100
    print(f'Novo salario: {salario+reajuste:.2f}\nReajuste ganho: {reajuste:.2f}\nEm percentual: 4 %')